haraka-aesni/merkle.h

```diff
@@ -0,0 +1,17 @@
+#if !defined( MERKLE_H_ )
```

Comment:
As for why I put merkle_sign into wots.c, well, that's because it was pretty tied into wots_gen_leafx*, which does most of the work.  I have since exported the API to wots_gen_leafx* (so performance testing could call it directly), so moving these two functions into merkle.c is reasonable - I'll go ahead and do that.

Reply:
Ok, I moved merkle_sign, merkle_gen_root into a new merkle.c .  Now, the haraka-aesni implementation should be a symlink to the shake256-avx2 implementation; I'll do that when I master the deep art of git symlinks...